src/nlr/NetworkLevelReasoner.cpp

```diff
@@ -289,6 +290,26 @@ void NetworkLevelReasoner::dumpTopology() const
         layer.second->dump();
 }
 
+void NetworkLevelReasoner::checkBoundsViolations()
```

Comment:
As mentioned in previous comment, this method would not be necessary.

Reply:
removed.